src/common/snap.F90

```diff
@@ -1983,6 +1988,7 @@ subroutine read_inputfile(snapinput_unit)
         end if
         nlevel = size(klevel)
         nk = nlevel
+        write (error_unit, *) "autodetection of grid-param: ", gparam
```

Comment:
Definitely not. All proper output is written to the netcdf-file. All output on the command-line is debug output, and all should be written to the same file-handle, otherwise output don't appear in the same order. Since error_unit is non-buffered, it is much better to use. Otherwise, people might not see the output before the program is finished. (Depending on the terminals buffer-settings.)

Reply:
Most buffer implementations flush on newline, but I can't find anything in the Fortran standard that requires this of the processor.

Reply:
Thought that, too. But the snap output from bionic did never appear in the meteorologists console. Using error_unit solved that.

Reply:
An alternative would be to force output to be line-buffered in the python code, but this solution is a bit easier :+1: